docs/topics/caching-mission-basic-interaction.adoc

```diff
@@ -0,0 +1,19 @@
+= Interacting with the Unmodified Booster
+
+. Navigate to the `greeting` service using your browser
+
+. Click _Greet_ once.
++
+Notice the `duration` value is above `2000`. Also notice _Cache state_ has changed form `Not cached` to `Cached`.
+
+. Wait 5 seconds and notice _Cache state_ has changed back to `Not cached`.
++
+The `TTL` for the cached value is set to 5 seconds.
```

Comment:
I think the TTL is actually 10 seconds.

Reply:
Please disregard this comment. The TTL is indeed 5 seconds and I got confused by the Spring Boot booster.